support-frontend/assets/components/headers/links/links.jsx

```diff
@@ -73,14 +80,15 @@ const Links = ({ location, getRef, countryGroupId }: PropTypes) => (
         }
         return true;
       }).map(({
-        href, text, trackAs, opensInNewWindow,
+        href, text, trackAs, opensInNewWindow, additionalClasses,
         }) => (
           <li
-            className={
+            className={[
               classNameWithModifiers(
                 'component-header-links__li',
                 [window.location.href.endsWith(href) ? 'active' : null],
-              )
+              ), (additionalClasses || null),
+            ].join(' ')
```

Comment:
I think you could use the classnames utility we import to just do something like:

```
import 'cx' from 'classnames'

...

className={cx(
  classNameWithModifiers(
    ...
  ), additionalClasses)
}
```